plugin/storage/integration/cassandra_test.go

```diff
@@ -134,4 +138,5 @@ func TestCassandraStorage(t *testing.T) {
 	require.NoError(t, s1.initializeCassandra())
 	// TODO: Support all other tests.
 	t.Run("GetDependencies", s1.testCassandraGetDependencies)
+	s1.IntegrationTestAll(t)
```

Comment:
oh, wow, we never tested Cassandra?

Reply:
yes,`IntegrationTestAll` was never called for cassendra